script.js

```diff
@@ -1,12 +1,377 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
+
```

Comment:
Have you set up the `prettier` plugin in vscode? I noticed that you're missing semi-colons at the end of some of your statements. This plugin would help to fix that for you
https://codeyourfuture.slack.com/archives/C01F7MF2FM2/p1617300742154200

Reply:
Hi @louisechow , I have a the `prettier` plugin installed and checked my settings if that is the case, which it is, but it still does not add semicolons or give me any error when saving.